packages/renderer/MdToHtml/linkReplacement.ts

```diff
@@ -36,14 +37,15 @@ export default function(href: string, options: Options = null): LinkReplacementR
 
 	const resourceHrefInfo = urlUtils.parseResourceUrl(href);
 	const isResourceUrl = options.resources && !!resourceHrefInfo;
+	const isExternalFile = href.startsWith('file:///') || href.startsWith('file://');
```

Comment:
Doesn't the 2nd part in this condition includes the 1st?

Reply:
Sorry, my bad.